.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: integration-tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Should we be adding a path on these to only run in the tools directory? Or can changes in other parts of the app break the tooling?

Reply:
That's a good idea, but we should also also have the generated pb.go files (i.e., `pkg/api/aggregation/v1/aggregation.pb.go`) there, just to catch the case of someone messing with the proto definitions.